endpoints/getting-started/openapi.yaml

```diff
@@ -6,6 +6,9 @@ info:
   version: "1.0.0"
 host: "echo-api.endpoints.YOUR-PROJECT-ID.cloud.goog"
 # [END swagger]
+# For App Engine deployments, delete the above "host:" line and remove the "# "
```

Comment:
The host changes when using App Engine, as the API is served from PROJECT-ID.appspot.com instead of *.PROJECT-ID.cloud.goog. This change is to make it easier for the user to follow the getting started guide for Endpoints on App Engine.